project_one.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="project_one.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <title>Project One</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="card">
+        <section class="snap">
+          <img src="./images/Snap.png" alt="img" />
+        </section>
+        <section class="main">
+          <h1>Tonic</h1>
+          <div class="mid">
+            <span class="can"> CANOPY </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> BACK END DEV </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> 2015 </span>
+          </div>
+          <p class="para">
+            A daily selection of privately personalized reads; no accounts or
```

Comment:
- [ ] Please use the same design for each card so it matches exactly the card in [Fegma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

- [ ] Please the card background color is different than the hole container background color fix that.
- [ ] The border of the card should match the one in Figma design 

![image](https://user-images.githubusercontent.com/74744787/159127204-35b076c2-e5be-40d1-b2c9-8076945025bf.png)

![image](https://user-images.githubusercontent.com/74744787/159127216-2a166df0-7581-427e-b8b6-17f0d51b8b54.png)

- [ ] Please Fix the padding of the button and make sure it matches the one in Figma design